website/docs/guides/1.15.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 1.15.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-1150-upgrade-guide"
+description: |-
+MongoDB Atlas Provider 1.15.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.15.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.15.0 has a number of new and exciting features.
+
+**New Resources, Data Sources, and Features:**
+
+
+**Breaking Changes:**
+
+Format of `okta_idp_id` that contains the IdP Id in `federated_settings_identity_provider` resource now accepts a different format to align with Atlas API. Both the current and new IdP Id format are accepted in terraform-provider 1.15.0 version.
+
+***WARNING:*** Old IdP Id format will no longer be accepted starting 1.16.0 and onwards. We recommend to update to the new format as soon as possible. A warning will appear if old Id is still being used. Follow the guide below to start using the new Id format.
+
+The old format was a 20-hex characters long string.
+
+The new format is a 24-hex characters long string.
+
+Update guide: 
+
+1. Get new 24-hex characters string ID value
+	- Option 1: Use Atlas API v20231001, both of those endpoints include the new `id` in the response:
+		- Use [Return all identity providers from the specified federation](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/2023-10-01/#tag/Federated-Authentication/operation/listIdentityProviders) endpoint
+		- Use [Return one identity provider from the specified federation by id](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/2023-10-01/#tag/Federated-Authentication/operation/listIdentityProviders) endpoint
+	- Option 2: Use `mongodbatlas_federated_settings_identity_providers` resource: 
+	```
+	data "mongodbatlas_federated_settings_identity_providers" "identity_provider" {
+  		federation_settings_id = <OLD_ID>
+	}
+	```
+	This will return the list of `mongodbatlas_federated_settings_identity_providers` with both `okta_idp_id` (20-hex characters Id) and `idp_id` (24-hex characters Id)
+3. Replace all occurrences of old Id, such as `identity_provider_id` in `mongodbatlas_federated_settings_identity_provider`
```

Comment:
From my understanding the singular data source is the only one that receives `identity_provider_id` as an argument, meaning users define it in their configuration and will need to replace the value. Might be worth clarifying.

Reply:
Clarified the docs